app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +20,11 @@
             </intent-filter>
         </activity>
         <activity android:name=".DetailActivity"/>
+
+        <service
+            android:name=".DownloadService"
+            android:enabled="true"
+            android:exported="true" />
```

Comment:
je potrebne, aby service bola exported? povazuje sa to za security hole, ak nie je potreba ju volat z inej appky